src/platform/Darwin/PlatformManagerImpl.cpp

```diff
@@ -54,8 +54,10 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack()
 
     mRunLoopSem = dispatch_semaphore_create(0);
 
+#if !CHIP_SYSTEM_CONFIG_USE_LIBEV
```

Comment:
This part is still not clear to me.  If we are here (or more to the point if the System::Layer will look for this queue), then CHIP_SYSTEM_CONFIG_USE_DISPATCH is defined, right?  But that's mutually exclusive with CHIP_SYSTEM_CONFIG_USE_LIBEV.  So why do we need the conditioning here?

Reply:
If we get here, we're on Darwin, which does have dispatch and *could* provide a queue to the system layer. But when we're compiling with `CHIP_SYSTEM_CONFIG_USE_LIBEV` (which is possible even under Darwin),  the system layer does *not* have queue support, and does not have the `SetDispatchQueue` method.

For clarity, I guess it would be better to just use `#if CHIP_SYSTEM_CONFIG_USE_DISPATCH`here instead of `#if !CHIP_SYSTEM_CONFIG_USE_LIBEV`. Under normal Darwin builds, that conditional is redundant, but it allows to use other mainloops *such as* the libev based one. Agree?

Reply:
BTW: I fully agree this be better a "POSTPONED past 1.0" thing, that's why I posted it as a draft in the first place. I'm happy though about you looking at it right now, and I'm using and testing it daily in my setup right now, so by the time it gets a regular PR it should be reasonably mature :-)

Reply:
> For clarity, I guess it would be better to just use #if CHIP_SYSTEM_CONFIG_USE_DISPATCHhere

Yes, that would be much more clear, since that's the thing we really care about: that the `SetDispatchQueue` method exists.  OK.